src/genericsite/templatetags/genericsite.py

```diff
@@ -99,6 +99,20 @@ def menu_aria_current(context, menuitem: str):
     return ""
 
 
+@register.simple_tag(takes_context=True)
```

Comment:
I didn't know you could encapsulate logic like this in template tags and it is Very Useful to know.